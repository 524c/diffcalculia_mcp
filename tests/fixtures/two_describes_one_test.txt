describe('Some Component', () => {
  describe('stuff()', () => {
    it('should do stuff', () => {
      blah.stuff();
    });
  });
});
