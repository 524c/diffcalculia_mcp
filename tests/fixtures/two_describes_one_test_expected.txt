describe('Some Component', () => {
  describe('stuff()', () => {
    it('should do stuff', () => {
      blah.stuff();
    });
  });

  describe('other_stuff()', () => {
    it('should do other stuff', () => {
      blah.other_stuff();
    });
  });

});
